src/components/Icons/Icon.tsx

```diff
@@ -82,6 +96,10 @@ export default function Icon(props: React.PropsWithChildren<IconProps>) {
     modifiers.push(iconRotation);
   }
 
+  if (color) {
+    modifiers.push(color);
+  }
+
   const iconProps = {
```

Comment:
One thing you have to watch out for here is that every time it re-renders, it's going to push that value to the modifiers string and eventually you will end up with something like this:
![image](https://user-images.githubusercontent.com/6432596/113346438-d7d14a80-9301-11eb-9568-1d404c314db5.png)